fenix/app/src/main/java/org/mozilla/fenix/components/appstate/AppAction.kt

```diff
@@ -208,4 +209,11 @@ sealed class AppAction : Action {
          */
         object PauseAction : AppLifecycleAction()
     }
+
+    /**
+     * State of standard error snackBar has changed.
+     */
+    data class ShowStandardSnackbarErrorAction(
```

Comment:
Maybe `UpdateStandardSnackbarErrorAction`? As the description mentions we are only update the value of the standard error snackBar.

Reply:
Done